flutter_mobx/lib/src/observer.dart

```diff
@@ -50,7 +53,14 @@ class _ObserverState extends State<Observer> {
   void initState() {
     super.initState();
 
-    _reaction = widget.createReaction(_invalidate);
+    _reaction = widget.createReaction(_invalidate, onError: (e, _) {
+      // If we're receiving a FlutterError, it's occurring internally in the
+      // setState() call. As such, we throw the exception so that it can be
+      // routed through Flutter's error handling machinery.
+      if (e is FlutterError) {
```

Comment:
Why only FlutterError? Why not Error?

Reply:
Why not:

```suggestion
    FlutterError.reportError(FlutterErrorDetails(
      library: 'mobx',
      exception: e,
      stack: stacktrace,
    ));
```

Reply:
Nice! We'll probably want that original stack if we've got it though.

Reply:
Suggestion accepted. Thanks.